queries/course_requests.js

```diff
@@ -1,31 +1,79 @@
 /**
- * This file holds the code of all queries related to the CourseRequests table.
- *
+ * This file holds the code of all queries related to the courseRequests table.
+ * 
  * @version 13/09/2022
  */
 module.exports = {
-  /**
-   * Get the total number of course requests in the database.
-   *
-   * A status of 200 is given for successful requests. A status of 400 for unsuccessful ones.
-   *
-   * @param {*} sql The mssql instance connected to the database currently used by the API.
-   * @param {*} res The object to send result to a given query sender.
-   */
-  getNumberOfCourseRequests: function (sql, res) {
-    const request = new sql.Request();
-
-    request.query(
-      "SELECT COUNT(ID) FROM CourseRequests WHERE Status=0",
-      function (err, recordset) {
-        if (err) {
-          res.status(400).json({ error: err });
+
+    /**
+     * Retrieves all the new course requests (status = 0)
```

Comment:
Here would be nice to mention the fact that it also updates the course requests when their retrieval was successful